requirements.txt

```diff
@@ -1,4 +1,4 @@
 cryptography==36.0.2
 pynacl==1.5.0
 mnemonic==0.20
-multiversx_sdk_core>=0.6.0, <0.7.0
+multiversx_sdk_core>=0.7.0,<0.8.0
```

Comment:
Forces users to upgrade their sdk-core, indeed.

We'll have less issues of this kind after merging the repositories.

Reply:
We should bump to minor version.